src/io/flutter/preview/PreviewView.java

```diff
@@ -766,14 +766,17 @@ Icon getFlutterDecoratedIcon() {
 
     LayeredIcon decorated = flutterDecoratedIcons.get(icon);
     if (decorated == null) {
-      final Icon prefix = FlutterIcons.Flutter;
+      final Icon badgeIcon = FlutterIcons.Flutter_badge;
 
       decorated = new LayeredIcon(2);
-      decorated.setIcon(prefix, 0, 0, 0);
-      decorated.setIcon(icon, 1, prefix.getIconWidth(), 0);
+      decorated.setIcon(icon, 0, 0, 0);
+      decorated.setIcon(badgeIcon, 1,
+                        icon.getIconWidth() + (icon.getIconWidth() - badgeIcon.getIconWidth()) / 2,
```

Comment:
What `(icon.getIconWidth() - badgeIcon.getIconWidth()) / 2` does?
It looks like we need a whitespace between these two items, but the way we compute is peculiar :-)

Reply:
It's a 8x8 icon centered in a 16x16 area (as if there were two 16x16 icons right next to each other).

Reply:
Ah, makes sense.
Thanks.